internal/files/repository.go

```diff
@@ -5,11 +5,33 @@
 package files
 
 import (
+	"bytes"
 	"fmt"
 	"os"
 	"path/filepath"
+
+	"gopkg.in/yaml.v3"
 )
 
+func isGitWorktree(path string) error {
```

Comment:
Nit. It would make sense for this function to return a boolean telling if this is a git worktree.
```suggestion
func isGitWorktree(path string) (bool, error) {
```

Reply:
In this case, would the boolean returned be the same as the error? Currently, if the path is not a git worktree folder it returns an error.